pkgs/os-specific/linux/framework-laptop-kmod/default.nix

```
@@ -7,13 +7,13 @@
 
 stdenv.mkDerivation rec {
   pname = "framework-laptop-kmod";
-  version = "unstable-2023-12-03";
+  version = "unstable-20240102";
```

```suggestion
  version = "0-unstable-20240102";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming